go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/foundriesio/fioconfig
 
-go 1.17
+go 1.19
```

Comment:
i believe the golang version included in v90 is 1.17.13. Not sure we can build this in meta-lmp yet.

Reply:
nvm